NanaZip.UI.Classic/SevenZip/CPP/7zip/UI/FileManager/PanelFolderChange.cpp

```diff
@@ -72,10 +72,73 @@ static bool DoesNameContainWildcard_SkipRoot(const UString &path)
 {
   return DoesNameContainWildcard(path.Ptr(NName::GetRootPrefixSize(path)));
 }
+// **************** NanaZip Modification Start **************
```

Comment:
Add an empty line before NanaZip Modification Start in this scenario for better git diff result.